android/app/src/main/res/layout/activity_main.xml

```diff
@@ -18,17 +46,39 @@
         app:layout_constraintTop_toTopOf="parent"
         tools:layout="@layout/fragment_map" />
 
+    <HorizontalScrollView
+        android:id="@+id/horizontal_scroll_view_container_filter"
+        style="@style/Theme.EcoMap"
```

Comment:
Is this needed?

Reply:
Not really, just a leftover from testing the style.
I will remove that